mvp-spec/ad-auction.md

```diff
@@ -1,25 +1,117 @@
-# Publishers requirements
+# Ad auction
 
 ## Goal of the document
 
-This document describes what is required for a Publisher to participate in the Prebid Addressability Framework (PAF).
+This document describes the Prebid Addressability Framework (PAF) requirements related to an ad auction.
 
 ## Overview
 
-The Prebid Addressability Framework enhances ad slots selling by instantiating an Audit Log. This Audit Log requires the following elements, that Publishers must implement:
-* a **Seed**, identifying a bid request for an ad slot for a given user
-* **Transmission Requests**, identifying the Seed and the requester at each step of the bidding workflow
-* **Transmission Responses**, as answers to Transmission Requests
-* The **Audit Log** itself, identifying the entities have been involved in the display of an ad
+The Prebid Addressability Framework enhances ad auctions by instantiating an **Audit Log**, built upon the following elements:
+* **Seed**
+* **Transmission Requests**
+* **Transmission Responses**
+* **Transmission Results**
 
-Seeds, Transmission Requests, and Transmissions Responses are signed.
+### Seed
 
-Publishers are also required to:
-* Expose an Identity endpoint, so that participants can verify the the publisher's signature.
+A Seed links together the publisher site identity and a transaction id. 
 
-## Selling ad slots with the Prebid Addressability Framework
+### Transmission Requests
 
-### ad slots, Seeds, and Transmissions Requests
+A Transmission Request links together a Seed, a sender identity, and an array of Transmission Results (see below). 
+
+Participants sending User Id and Preferences with bid requests must include a Transmission Request alongside that communication.
+
+### Transmission Responses
+
+A Transmission Response links together a receiver identity, one or several transaction id, one or several content id, and a tree of Transmission Resulsts (see below). 
```

Comment:
Do you think that we would be more descriptive of the functional aspects for all those definitions?

Example for the Transmission Response:
A Transmission Response is a response of a Receiver to a Sender for PAF Data. It expresses the commitment of the Receiver to respect the Preferences of the User for the given Identifiers thanks to a signature. Its contains:
* the identity of the Receiver
* the status of the Transmission (Equal 'Success' if the Receiver commits to use correctly the PAF Data).
* zero to many associations between a Transaction Id and Content Id depending if the Receiver offers potential Addressable Contents itself.
* A tree of Transmission Results that reflects the Transmissions instantiated thanks to the Receiver and that can participate in the Addressable Content.

Reply:
Yes, I think it's much better. 

FYI, I removed from the glossary:
- Receiver and Sender, as these terms are self-explanatory. I use them without capital letters if need be.
- PAF Data, replaced by User Id and Preferences, which is more self-explanatory.
- Addressable Content, replaced by "ad" or "content" for the same reasons

Transaction Id and Content Id are also not in the Glossary as of now, but they may have to be added.

Reply:
I updated the description of Seed, Transmission Request, Transmission Response